tests/gpu/misc/nvblas.conf

```diff
@@ -0,0 +1,3 @@
+NVBLAS_LOGFILE /var/log/nvblas.log
```

Comment:
I won't need it once the change to config_R_cuda.sh makes it into the build process, but will until then because I'm pulling the built containers.
I can remove it from the PR though to reduce clutter.

Reply:
It's actually a little difficult to remove it before the change to config_R_cuda.sh has been deployed.  I would have to remove it from my Ansible and Docker code as well.  After this has been deployed I can easily remove all references to it.  Would that be okay?